### PR TITLE
Apply ESLint to the project

### LINKED ISSUE #71: Apply ESLint to the project
- [x] Add eslint (already done)
- [x] Add datarockets-eslint rules  (already done)
- [x] Add eslint to the Github Actions
- [x] Refactor `specs` folder just for check that everything is fine with linter

### PR DESCRIPTION
**Pull request by npupko**
Close #71

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -13,3 +13,12 @@ jobs:
...
           npm install
           make test
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Run ESLint
+        run: |
+          npm install
+          ./node_modules/.bin/eslint .
```
Comment (lazycoder9) [inline .github/workflows/main.yml]: @npupko what about writing make command? Or at last we can write new command in `script` section of `package.json`

**Review by lazycoder9 — DISMISSED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (npupko) [reply .github/workflows/main.yml]: @lazycoder9 Yep, thanks.

P.S.: Today I learned what is `.PHONY`:)

**Comment by npupko** [comment]
The checks are going to pass when I will merge [this PR](https://github.com/datarockets/mona/pull/70)

**Comment by npupko** [comment]
Have finished work on it

File: Makefile [hunk 2]
```diff
@@ -1,4 +1,7 @@
 test:
 	bin/mocha specs
 
-.PHONY: test
+eslint:
+	node_modules/.bin/eslint .
+
+.PHONY: test eslint
```
Comment (lazycoder9) [inline Makefile]: `eslint` is not necessary in PHONY, because it is not a special command for make utility

**Review by lazycoder9 — CHANGES_REQUESTED** [review]

**Comment by lazycoder9** [comment]
@npupko also, I saw that file renamed from camel case to snake case (waterOrderSpec -> water_order_spec). I thought that in JS community we should name files in camel case too

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -13,3 +13,12 @@ jobs:
...
           npm install
           make test
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Run ESLint
+        run: |
+          npm install
+          make eslint
```
Comment (lazycoder9) [inline .github/workflows/main.yml:23]: Maybe we can run both eslint and tests in one job?

**Review by lazycoder9 — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 3 above)
Comment (npupko) [reply .github/workflows/main.yml:23]: @lazycoder9 What's the point of it? 
I see many pros of separated jobs:
1. We have different results for different checks. (broken tests and lints separately)
2. We run jobs in parallel

Comment (lazycoder9) [reply .github/workflows/main.yml:23]: @npupko maybe it is not so bad thing, but I thought about running `npm install` twice

**Review by lazycoder9 — APPROVED** [review]

File: .github/workflows/main.yml (hunk 3 above)
Comment (lazycoder9) [reply .github/workflows/main.yml:23]: @npupko anyway, we can accept it now, because it is not a big deal. Approved

Comment (npupko) [reply .github/workflows/main.yml:23]: @lazycoder9 Yeah, I have thought about it but don't found any solution for now.
GitLab has `before_script` section and `stage: build` for it.

**Pull request merged**